apps/discord/win.py

```diff
@@ -82,3 +89,23 @@ def discord_answer_call():
 
     def discord_decline_call():
         actions.key("esc")
+
+    def discord_go_current_call():
+        actions.key("ctrl-shift-alt-v")
+
+    def discord_toggle_dms():
+        actions.key("ctrl-alt-right")
+
+    def discord_quick_switcher(dest_type: str, dest_search: str):
+        actions.key("ctrl-k")
+        if dest_type:
+            if dest_type == "user":
+                actions.key("@")
+            if dest_type == "channel":
+                actions.key("#")
+            if dest_type == "voice":
+                actions.key("!")
+            if dest_type == "server":
+                actions.key("*")
```

Comment:
We could put this mapping in the list above instead.

Reply:
yeah that's a good idea, it's in the list now